src/modules/invoice-accounts/jobs/update-customer.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const { get } = require('lodash');
+const { updateCustomer } = require('../../../lib/connectors/charge-module/customers-with-retry');
+const JOB_NAME = 'customer.updateAccount.*';
+
+const createMessage = async (invoiceAccountId) => {
+  return {
+    name: JOB_NAME.replace('*', invoiceAccountId),
+    data: {
+      invoiceAccountId
+    },
+    options: {
+      singletonKey: invoiceAccountId,
+      retryLimit: 3,
+      retryDelay: 30,
+      retryBackoff: true
+    }
+  };
+};
+
+const handleUpdateCustomer = async job => {
+  const invoiceAccountId = get(job, 'data.invoiceAccountId');
+  return updateCustomer(invoiceAccountId);
+};
+
```

Comment:
Doesn't look like we have an on-complete/failed handler for this.  Just wondering what we should do if even after x attempts, the CM still does not respond.

We should probably think about the best option for handling this.

Reply:
Yeah, I agree - We don't currently have a defined approach for handling that case. 

For now, I'll add on-complete and failed handlers which will log an event to the console. 

More than happy to add additional functionality, but I'm not sure what that functionality would be. I thought about adding an email notification, similar to what we get from FME. E.g. "This particular address could not be updated in the charge module" - but I didn't go ahead because I don't know if that's actually helpful/desirable.